packages/renderers-js/src/utils/linkOverrides.ts

```diff
@@ -49,13 +49,20 @@ export function getImportFromFactory(
         resolvers: overrides.resolvers ?? {},
     };
 
+    // todo: figure out if this logic makes sense, and if so apply to all the other 'ProgramNode-able" nodes
+    const handleProgramLink = (node?: ProgramLinkNode): string | undefined => {
+        if (!node) return;
+        const programName = node.name;
+        return linkOverrides.programs[programName];
```

Comment:
Hmm I'm not sure if this is what we want since we'll end up linking a "program" override on a "defined type" override.

Reply:
Ah yes I meant to do it in the reverse. The definedTypes override should be taking precedent here.